### PR TITLE
Fixes #12434: Graphs home page

### PR DESCRIPTION
**Pull request by Haypierre**
https://www.rudder-project.org/redmine/issues/12434

### CODE REVIEW

**Comment by Haypierre** [comment]
PR rebased

**Comment by Haypierre** [comment]
PR rebased

**Comment by Haypierre** [comment]
Commit modified

File: rudder-web/src/main/resources/version.properties [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # Major.minor without micro
 rudder-major-version=${rudder-major-version}
 # Full version, with SNAPSHOT etc
-rudder-full-version=${version}
+rudder-full-version=4.3
```
Comment (VinceMacBuche) [inline rudder-web/src/main/resources/version.properties]: You should not commit this file, please revert change on this file

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/HomePage.scala [hunk 2]
```diff
@@ -283,6 +288,19 @@ class HomePage extends Loggable {
...
+    val osTypes = AixOS       ::
+      SolarisOS               ::
+      BsdType.allKnownTypes   :::
+      LinuxType.allKnownTypes :::
+      WindowsType.allKnownTypes
+
+    // mapping between os name and their string representation (printed on screen).
+
+    val osNames = JsObj(osTypes.map {os =>
+      (S.?("os.name." + os.name), Str(os.name))}:_*)
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/HomePage.scala]: Put it on one line or do 
<pre>
JsObj(osTypes.map {
            os => (S.?("os.name." + os.name), Str(os.name))}:_*)
          }:_*)
                  
</pre>

**Comment by Haypierre** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
To me it's ok, I'll let françois validate his review :)

**Pull request merged**